langchain/llms/utils.py

```diff
@@ -0,0 +1,8 @@
+"""Common utility functions for working with LLM APIs."""
+import re
+from typing import List
+
+
+def enforce_stop_tokens(text: str, stop: List[str]) -> str:
```

Comment:
Double checking - the prompt isn't returned as part of the generated text, right?

Reply:
no it shouldnt be